tests/llm/test_openai_oauth_subscription_flow.py

```
@@ -1379,6 +1380,42 @@ async def test_oauth_codex_partial_text_without_completed_is_disconnected(
     assert lifecycle.error.retryable is True
 
 
+@pytest.mark.asyncio
+async def test_oauth_codex_stream_transport_error_records_disconnected_lifecycle(
+    monkeypatch: pytest.MonkeyPatch,
+) -> None:
+    _install_oauth_codex_test_auth(monkeypatch)
+
+    class _BrokenStreamResponse(_FakeResponse):
+        async def aiter_lines(self):  # type: ignore[no-untyped-def]
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `aiter_lines`

(ANN202)